phpunit.xml

```diff
@@ -101,5 +101,10 @@
     <testsuite name="CorporateBranding">
       <directory>src/plugins/orangehrmCorporateBrandingPlugin/test</directory>
     </testsuite>
+    <testsuite name="Dashboard">
+      <directory>src/plugins/orangehrmDashboardPlugin/test/Service</directory>
+      <directory>src/plugins/orangehrmDashboardPlugin/test/Dao</directory>
+      <directory>src/plugins/orangehrmDashboardPlugin/test/Api</directory>
```

Comment:
```suggestion
      <directory>src/plugins/orangehrmDashboardPlugin/test</directory>
```
if the dashboard test directory does not have legacy code, you can do this

Reply:
ah ok I saw in CorporateBranding but was not sure the reason. changed it